### PR TITLE
The `replace` property is not mandatory anymore (#57)

### PR DESCRIPTION
**Pull request by LasaleFamine**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -133,14 +133,10 @@ class Kubozer {
...
 
-    if (optionJS.files === undefined && optionCSS.files === undefined) {
-      throw new Error('WARNING REPLACE(): replace method called but "files" not found in configuration');
-    }
-
     return new Promise((resolve, reject) => {
       try {
-        const changedCSS = replaceInFile.sync(optionCSS);
-        const changedJS  = replaceInFile.sync(optionJS);
+        const changedCSS = replaceInFile.sync(optionCSS.files ? optionCSS : defaultOptions);
```
Comment (StefanoMagrassi) [inline src/index.js:138]: @LasaleFamine maybe you can use Ramda's [merge](http://ramdajs.com/docs/#merge) method

File: src/index.js [hunk 2]
```diff
@@ -133,14 +133,10 @@ class Kubozer {
...
+        const changedJS  = replaceInFile.sync(optionJS.files ? optionJS : defaultOptions);
```
Comment (StefanoMagrassi) [inline src/index.js:139]: @LasaleFamine same as previous comment

File: test/index.test.js [hunk 3]
```diff
@@ -115,6 +115,18 @@ test('replace() products correct output', async t => {
   t.is(fileReplace, `<!DOCTYPE html><html><head>\n    <meta charset=\"utf-8\">\n    <title></title>\n\n      \n        <link rel=\"stylesheet\" href=\"/assets/style.css\">\n        \n\n      \n        <script src=\"/assets/javascript.js\"></script>\n        \n\n  </head>\n  <body>\n\n  \n\n</body></html>`);
 });
 
+test('when running replace() the "replace" property of configuration is not mandatory', async t => {
+  const {tmpDir, webpackConfig} = t.context;
+  const config                  = hconf(tmpDir, ['FOLDERS', 'VULCANIZE_NO_JS', 'MANIFEST']);
```
Comment (StefanoMagrassi) [inline test/index.test.js]: @LasaleFamine maybe 'MANIFEST' configuration is useless here

File: src/index.js (hunk 1 above)
Comment (LasaleFamine) [reply src/index.js:138]: As discuss, we can introduce new logic (and libs) during the refactor.

File: test/index.test.js (hunk 3 above)
Comment (LasaleFamine) [reply test/index.test.js]: Now should be correct.

**Pull request merged**